PennMobile/Weekly Schedule/WeeklyScheduleViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ScheduleViewController.swift
+//  PennMobile
+//
+//  Created by CHOI Jongmin on 14/2/2020.
+//  Copyright © 2020 PennLabs. All rights reserved.
+//
+
+import UIKit
+
+protocol WeeklyScheduleCellDelegate: ModularTableViewCellDelegate, BuildingMapSelectable {}
+
+final class WeeklyScheduleTableViewModel: ModularTableViewModel {}
+
+class WeeklyScheduleViewController: GenericViewController, IndicatorEnabled, ShowsAlert {
```

Comment:
Let's change the class and file names to "CourseScheduleViewController" to be more descriptive.

Reply:
Yup, I just finished the change. Would you like me to change the title of the viewcontroller to also be **Course Schedule**, or would you like it to stay as **Weekly Schedule**?

Reply:
Let’s change everything to CourseSchedule